client/blocks/import/capture/capture-input.tsx

```diff
@@ -15,15 +15,16 @@ import type { OnInputChange, OnInputEnter } from './types';
 import type { FunctionComponent, ReactNode } from 'react';
 
 interface Props {
-	onInputEnter: OnInputEnter;
-	onInputChange?: OnInputChange;
-	onDontHaveSiteAddressClick?: () => void;
+	dontHaveSiteAddressLabel?: string;
 	hasError?: boolean;
-	skipInitialChecking?: boolean;
+	hideImporterListLink?: boolean;
 	label?: ReactNode;
+	nextLabelText?: string;
+	onDontHaveSiteAddressClick?: () => void;
+	onInputChange?: OnInputChange;
+	onInputEnter: OnInputEnter;
 	placeholder?: string;
-	dontHaveSiteAddressLabel?: string;
-	hideImporterListLink?: boolean;
+	skipInitialChecking?: boolean;
```

Comment:
It took some time for me to find the difference 😅, so I sum up for anyone who reads later:
- properties ordered alphabetically,
- `nextLabelText` property added.